lib/langchain/loader.rb

```diff
@@ -90,7 +90,7 @@ def load(&block)
     private
 
     def load_from_url
-      URI.parse(@path).open
+      URI.parse(URI::DEFAULT_PARSER.escape(@path)).open
```

Comment:
@michelson Think we could add a quick spec for this?

Reply:
spec added, not sure if that tests the whole thing,but I have the fork in production and works